packages/editor/core/src/ui/hooks/useEditor.tsx

```diff
@@ -23,26 +27,44 @@ interface CustomEditorProps {
   forwardedRef?: any;
 }
 
-export const useEditor = ({ uploadFile, deleteFile, editorProps = {}, value, extensions = [], onChange, setIsSubmitting, debouncedUpdatesEnabled, forwardedRef, setShouldShowAlert, }: CustomEditorProps) => {
+export const useEditor = ({
+  uploadFile,
```

Comment:
use props as your function argument and de-struct it inside the function for better readability.